lua/chainsaw/log-statements-data.lua

```diff
@@ -16,15 +17,17 @@ return {
 	},
 	objectLog = {
 		nvim_lua = 'vim.notify("%s %s: " .. vim.inspect(%s))',
-		typescript = 'console.log("%s %s:", %s)',
-		javascript = 'console.log("%s %s:", JSON.stringify(%s))',
+		typescript = 'console.log("%s %s:", JSON.stringify(%s))',
+		typescriptreact = 'console.log("%s %s:", JSON.stringify(%s))',
+		javascript = 'console.log("%s %s:", JSON.ststringifyringify(%s))',
```

Comment:
"ststringifyringify", is a copypaste error, I assume?

Reply:
Yeah, I'm sorry.